database/migrations/2024_11_14_184759_add_ogl_question_to_manuscript_records_table.php

```diff
@@ -12,7 +12,7 @@
     public function up(): void
     {
         Schema::table('manuscript_records', function (Blueprint $table) {
-            $table->boolean('do_not_apply_ogl')->default(false);
+            $table->boolean('apply_ogl')->default(true);
```

Comment:
This is great but past migrations won't be applied to the staging and production environments. There's two ways to handle it, keep this here and manually change the DBs (which we can do here) or do a new migration that simply renames the column.

Reply:
We will do it manually this time.